src/pfe/file-watcher/server/src/projects/projectSpecifications.ts

```diff
@@ -550,7 +550,16 @@ const changeMavenProfiles = async function (mavenProfiles: any, operation: Opera
         return;
     }
 
-    if (mavenProfiles && !(mavenProfiles instanceof Array)) {
+    let arrayIsNotString = false;
+    if (mavenProfiles instanceof Array) {
+        mavenProfiles.forEach(function(item: any): void {
+           if (typeof item !== "string") {
+               arrayIsNotString = true;
```

Comment:
can probably break here since we only need one element to be not a string

Reply:
updated it